### PR TITLE
livewire:script and livewire:assets

### PR DESCRIPTION
**Pull request by ryanmitchell**
This PR adds support for {{livewire:script}}...{{/livewire:script}} and {{livewire:assets}}...{{/livewire:assets}}.

For @edalzell

Closes https://github.com/jonassiewertsen/statamic-livewire/issues/68

### CODE REVIEW

File: src/Tags/Livewire.php [hunk 1]
```diff
@@ -86,4 +86,37 @@ public function scriptConfig(): string
...
+     *
+     * {{ livewire:script }}...{{ /livewire:script }}
+     */
+    public function script(): void
+    {
+        $html = trim((string) $this->parse());
+        
+        $key = md5($html);
+        
+        if (in_array($key, \Livewire\Features\SupportScriptsAndAssets\SupportScriptsAndAssets::$alreadyRunAssetKeys)) {
```
Comment (edalzell) [inline src/Tags/Livewire.php:115]: why the empty `if`?

**Comment by edalzell** [comment]
Amazing!

File: src/Tags/Livewire.php (hunk 1 above)
Comment (ryanmitchell) [reply src/Tags/Livewire.php:115]: Just lifted it from the livewire code. Makes it easier to copy paste if it changes

Comment (edalzell) [reply src/Tags/Livewire.php:115]: That's not the code in the `scripts` code in LW though:

```php
        Blade::directive('script', function () {
            $key = static::getUniqueBladeCompileTimeKey();

            return <<<PHP
                <?php
                    \$__scriptKey = '$key';
                    ob_start();
                ?>
            PHP;
        });

        Blade::directive('endscript', function () {
            return <<<PHP
                <?php
                    \$__output = ob_get_clean();

                    \Livewire\store(\$this)->push('scripts', \$__output, \$__scriptKey)
                ?>
            PHP;
        });
```

Comment (ryanmitchell) [reply src/Tags/Livewire.php]: had them the wrong way round, thanks

**Comment by jonassiewertsen** [comment]
Thanks!

**Pull request merged**